tests/App/AppKernel.php

```diff
@@ -107,9 +95,4 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
             $loader->load($this->getProjectDir().'/config/config_sonata_block_v4.yaml');
         }
     }
-
-    private function getBaseDir(): string
-    {
-        return sys_get_temp_dir().'/sonata-page-bundle/var/';
```

Comment:
Never likes to use this `sys_get_temp_dir`

Reply:
Why ? It's done on other projects https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/4.x/tests/App/AppKernel.php#L107

Reply:
It's done on all sonata projects, I know.

Reply:
I probably have all /tmp folder full of sonata cacheses of my previous work on sonata

Reply:
tmp content is automatically delete (not sure when thought).

I would say it's better to avoid creating a directory in the project which will never be empty.

Reply:
O my system this method is returning different when calling it from cli and when it is through a webserver, do you know a workaround for that? I dont really mind to have a directory created on the project, it kinda belongs to the project, having it outside makes things like: lets do a rm on the dir somehow impossible unless you start dumping around

Reply:
With an example file: test.php
```
<?php
    echo sys_get_temp_dir() . PHP_EOL;
?>
```

You can run TMPDIR=/custom/location php test.php